seed/views/portfoliomanager.py

```diff
@@ -197,6 +226,7 @@ def get_list_of_report_templates(self):
                 _log.debug("Received the following child JSON return: " + json.dumps(child_object, indent=2))
                 for child_row in child_object:
                     child_row['z_seed_child_row'] = True
+                    child_row['display_name'] = "  -  %s" % child_row['name']
```

Comment:
This is the third important addition here.  This adds a prefix to the child templates and makes them look slick.  This should eliminate, or greatly reduce, any confusion between the template list on ESPM and the list on our dropdown.